build-system/test-server.js

```diff
@@ -111,5 +111,17 @@ app.use('/form/post', function(req, res) {
   });
 });
 
+/**
+ * Serve entry point script url
+ */
+app.get('/dist/ww.js', function(req, res, next) {
```

Comment:
In app.js, around line 850, mode is not set to `'default'` and so the min version is used. 

When I comment out the added code in test-server.js, test-bind-impl.js pulls in the min version or outright fails if `gulp dist` hasn't been run since the last `gulp clean`. Witht his code added, the test pulls in the max version.